ReactiveUI/ReactiveCommand.cs

```diff
@@ -281,7 +281,7 @@ public static ReactiveCommand<object> CreateCombined(params IReactiveCommand[] c
     /// </summary>
     public class ReactiveCommand<T> : IReactiveCommand<T>, IReactiveCommand
     {
-#if NET_45
+#if NET_40
```

Comment:
I thought this fork only purpose was to support net40.
If we need to support net45 as well, I can specify both, sure.